name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/util/chart/MeasurementTool.java

```diff
@@ -119,7 +120,7 @@ public void paintControl(PaintEvent e)
             var text = buffer.toString();
 
             Point txtExtend = e.gc.textExtent(text);
-            Rectangle plotArea = chart.getPlotArea().getClientArea();
+            Rectangle plotArea = ((Composite) chart.getPlotArea().getControl()).getClientArea();
```

Comment:
I think I made a mistake here : getControl then casting to Composite is one action too much ?

((Composite) chart.getPlotArea()) should be sufficient ?